app/models/loan.rb

```diff
@@ -64,6 +70,8 @@ class Loan < ActiveRecord::Base
   attr_option_settable :status, :loan_type, :project_type, :public_level
 
 
+  attr_custom_model_linkable LOAN_CRITERIA_FIELD_SET_SYMBOL, POST_ANALYSIS_FIELD_SET_SYMBOL
```

Comment:
The way I see it, these are essentially part of our schema, which we have to operate under the assumption will be, relatively speaking, lexically stable over most of the life of the software. The test suite should catch regressions in the unlikely event that we need to rename these, and since the constants contain the names of the symbols themselves, we're realistically going to need to rename those too if we don't want them to have misleading names for the rest of forever, which makes the addition of an extra constant into the process sort of redundant. `LOAN_CRITERIA_FIELD_SET_SYMBOL = :something_else_entirely` might save us a `find --> replace` in our editors, but I'm not sure of the benefit otherwise.